packages/backend/src/core/activitypub/ApRequestService.ts

```diff
@@ -204,8 +204,8 @@ export class ApRequestService {
 
 		//#region リクエスト先がhtmlかつactivity+jsonへのalternate linkタグがあるとき
 		const contentType = res.headers.get('content-type');
-		
-		if ((contentType === 'text/html' || contentType?.startsWith('text/html;')) && _followAlternate === true) {
+
+		if (contentType?.split(';')[0]?.trimEnd()?.toLowerCase() === 'text/html' && _followAlternate === true) {
```

Comment:
ちゃんとチェーンするなら
```suggestion
		if (contentType?.split(';')?.[0]?.trimEnd()?.toLowerCase() === 'text/html' && _followAlternate === true) {
```

だけど
```suggestion
		if (contentType?.split(';')[0].trimEnd().toLowerCase() === 'text/html' && _followAlternate === true) {
```

でも通りそう(未検証)

Reply:
null.split(';')[0] はundefined(TSに怒られたので直したからこう)

Reply:
undefinedになるのはそうなんだけど、()のあとだけ省略するのは不揃いな気がしてます

Reply:
`(contentType ?? '')` にするとかかしら

Reply:
やった